src/pages/Settings/Settings.tsx

```diff
@@ -154,6 +155,20 @@ class Settings extends Component {
                             />
                         </div>
                     </div>
+                    <div className="git">
+                        <div className="flex-row-box">
+                            <p>{t.t('page.settings.git.version', 'Version:')}</p>
+                            <p style={{ marginLeft: '0.4rem' }}>Alpha {version}</p>
```

Comment:
probably not the best way to hardcode "alpha". Probably move this to package.json

Reply:
It's not valid in package.json